src/core/core.scale.js

```diff
@@ -634,13 +634,26 @@ module.exports = Element.extend({
 		var cosRotation = Math.cos(labelRotationRadians);
 		var longestRotatedLabel = me.longestLabelWidth * cosRotation;
 		var result = [];
-		var i, tick, shouldSkip;
-
+		var i, tick;
+
+		// Specify the label spacing
+		var labelSpacing;
+		if (optionTicks.labelSpacing) {
+			labelSpacing = optionTicks.labelSpacing;
+			// If not an integer, reject
+			if (labelSpacing !== Math.floor(labelSpacing)) {
```

Comment:
I would just combine this if statement with the previous one:

    if (optionTicks.labelSpacing && optionTicks.labelSpacing !== Math.floor(optionTicks.labelSpacing)) {